### PR TITLE
Add my PhD thesis

### PR DESCRIPTION
**Pull request by ptersilie**

### CODE REVIEW

File: softdev.bib [hunk 1]
```diff
@@ -2385,3 +2385,11 @@ @article{tratt08domain
     pages = "1--40",
     publisher = "ACM"
 }
+
+@phdthesis{diekmann18editing,
+    author = {Lukas Diekmann},
+    title = {Editing composed languages},
+    year = {2018},
```
Comment (ltratt) [inline softdev.bib]: This should be the year / month you got the acceptance and the thing was formally published, so Jun 2019 I think? [I'm assuming you will hand the final copy in this month!]

Comment (ptersilie) [reply softdev.bib]: Ah, I thought it should be the date on the thesis, which is the due date. Do I need to change that too?

Comment (ptersilie) [reply softdev.bib]: Fixed.

**Comment by ltratt** [comment]
Please squash.

**Comment by ptersilie** [comment]
Squashed.

**Pull request merged**